src/Symfony/Component/Console/Descriptor/DescriptorInterface.php

```diff
@@ -21,7 +21,7 @@
 interface DescriptorInterface
 {
     /**
-     * Describes an object if supported.
+     * Describes an object if the class that the object is an instance of is supported.
```

Comment:
If we're improving this description, I'd love to improve it even further to explain things better.

1) *"Describes an object ..."*  What does "describe" an object mean? Does it return the main PHPdoc description of the class? Does it return an array of the names of the properties and methods of the object?

2) *"... if the object is supported."*  By who or how? This interface doesn't have a `supports()` method, so I don't understand how can I support or not support objects here.

Thanks!